lua/neorg/modules/core/norg/journal/module.lua

```diff
@@ -102,9 +102,9 @@ end
 module.private = {
     --- Opens a diary entry at the given time
     ---@param time number #The time to open the journal entry at as returned by `os.time()`
-    ---@param custom_date string #A YYYY-mm-dd string that specifies a date to open the diary at instead
+    ---@param custom_date string #(Optional) A YYYY-mm-dd string that specifies a date to open the diary at instead
```

Comment:
just use string|nil as the type and remove that optional

Reply:
or make it optional:
```suggestion
    ---@param custom_date? string #A YYYY-mm-dd string that specifies a date to open the diary at instead
```

Reply:
done!